docs/references/issue-rules/index.md

```diff
@@ -33,109 +38,109 @@ To see how issue rules are implemented, visit the [Nordcraft GitHub repository](
-| **Unknown context provider formula** | Error    |     No     | Finds references to un-exposed context formulas. This could happen if a context formula had previously been exposed in context, but was now no longer exposed.                                                                            |
-| **Unknown context provider**         | Error    |     No     | Finds references to context providers (components) that don't exist in the project.                                                                                                                                                       |
+| Rule                                                                                                                                                                         | Severity | Auto fixes | Description                                                                                                                                                                                                                               |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | :--------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **[No context consumption possible](https://github.com/nordcraftengine/nordcraft/blob/main/packages/search/src/rules/issues/context/noContextConsumersRule.ts)**             | Warning  |     No     | Flags components that expose formulas or workflows in context, but don't have a slot element or a component as a child. This would mean that it was not possible for other components/elements to consume the context formulas/workflows. |
+| **[Unknown context formula](https://github.com/nordcraftengine/nordcraft/blob/main/packages/search/src/rules/issues/context/unknownContextFormulaRule.ts)**                  | Error    |     No     | Finds references to unknown context formulas. This could happen if a context provider no longer had a formula by the name declared in the context consumer's reference.                                                                   |
+| **[Unknown context provider formula](https://github.com/nordcraftengine/nordcraft/blob/main/packages/search/src/rules/issues/context/unknownContextProviderFormulaRule.ts)** | Error    |     No     | Finds references to un-exposed context formulas. This could happen if a context formula had previously been exposed in context, but was now no longer exposed.                                                                            |
```

Comment:
```suggestion
| **[Unknown context provider formula](https://github.com/nordcraftengine/nordcraft/blob/main/packages/search/src/rules/issues/context/unknownContextProviderFormulaRule.ts)** | Error    |     No     | Finds references to un-exposed context formulas. This could happen if a context formula had previously been exposed in context, but is now no longer exposed.                                                                            |
```